about.html

```diff
@@ -0,0 +1,130 @@
+        <ul class="menu-list">
+          <li class="list"><a href="index.html">Home</a></li>
+          <li class="list"><a href="about.html"></a>About</li>
+          <li class="list"><a href="#services"></a>Services</li>
+          <li class="list"><a href="contact.html"></a>Contact</li>
+          <li class="list"><a href="#"></a>Extras</li>
+          <li class="list"><a href="#"></a>SignUp</li>
+        </ul>
+      </div>
+      <!-- Hamburger -->
+
+    </nav>
+    <div class="hamburger">
+      <div class="bar ham1"></div>
+      <div class="bar ham2"></div>
+      <div class="bar ham3"></div>
+    </div>
+  </header>
+
+  <main class="about">
+    <section class="hero" id="home">
+      <div class="top-txt">
+        <h3 class="hello-txt">
+          Hello! World of Digital Minds <br>
+        </h3>
+        <h2 class="title">
+          Geolex Cyber Space 2022 <br>
+          Protecting Data through knowledge sharing
+        </h2>
+        <p class="p2">
+          This year's webinar is focussing on how Big data has affected Data Security and Cyber crimes.
+          How can we protect ourselves from falling victim to cyber crime as active members of the digital community
+        </p>
+      </div>
+
+      <p class="address">please contact us per Email for furthure questions about Geolex Summit 2022 <br>
+        geolex@cyberspace.org
+      </p>
+
+      <div class="theme">
+        <h2 class="webinar-theme">
+          Geolex Cyber Space <br>
+          Digital Safe Webinar 2022 Logo
+        </h2>
+      </div>
+
+      <div class="webinar-logo">
+        <h2 class="logo2"><i class="fa-brands fa-microblog"> Geolex</i> <br> Cyber-space Webinar </h2>
+        <p class="details">18th-20th Oct 2015 |Lusaka, Zambia</p>
+      </div>
+
+    </section>
+    <section class="past-webinar">
+      <h2 class="see-past"> Highlights of our previous webinars</h2>
+
+      <p class="address">
+        Here are a few photos of our 2 previous webinars in Livingstone.
+      </p>
+
+      <div class="see-pix">
+        <h2 class="prev">2021</h2>
+        <p class="see-details">cyberspace webinar in Lusaka</p>
+      </div>
+      <div class="see-pix2">
+        <h2 class="prev">2020</h2>
+        <p class="see-details">cyberspace webinar in Kitwe</p>
+      </div>
+
+      <div class="partners desktop">
+        <h2 class="partner">
+          Partners
+        </h2>
+        <div class="partner-list">
+          <h2>ICTAZ</h2>
+          <h2>Microsoft</h2>
+          <h2><i class="fa-brands fa-squarespace">Airtel</i></h2>
+        </div>
+        <div class="partner-list2">
+          <h2><i class="fa-brands fa-hive"> Huawei</i></h2>
+          <h2>Digital Safe</h2>
+        </div>
+      </div>
+
+    </section>
+
+    <footer class="footer desktop">
+      <h2 class="footer-name">Geolex Cyber Space <br> Lusaka</h2>
+      <p class="footer-p">
+        2022 Geolex cyberspace <br>rights reserved
+      </p>
+    </footer>
```

Comment:
- [ ] As the guidelines indicate, the about page doesn't have a partners section, and the footer's background is dark for this page (only in the desktop version)

## Your design:
![image](https://user-images.githubusercontent.com/50721479/190704100-721dd90c-d315-4964-9c2a-681c690f2d3c.png)

## Guidelines: 
![image](https://user-images.githubusercontent.com/50721479/190704169-c20b5f51-9e1b-4611-8e6e-0da41f35969c.png)

Reply:
I have resolved this, thank you